src/slices/AccountSlice.ts

```diff
@@ -140,14 +159,14 @@ export const getBalances = createAsyncThunk(
     return {
       balances: {
         gohm: ethers.utils.formatEther(gOhmBalance),
-        ohm: ethers.utils.formatUnits(ohmBalance, "gwei"),
-        sohm: ethers.utils.formatUnits(sohmBalance, "gwei"),
+        ohmV1: ethers.utils.formatUnits(ohmBalance, "gwei"),
+        sohmV1: ethers.utils.formatUnits(sohmBalance, "gwei"),
         fsohm: ethers.utils.formatUnits(fsohmBalance, "gwei"),
         wsohm: ethers.utils.formatEther(wsohmBalance),
         fiatDaowsohm: ethers.utils.formatEther(fiatDaowsohmBalance),
-        wsohmAsSohm: ethers.utils.formatUnits(wsohmAsSohm, "gwei"),
         pool: ethers.utils.formatUnits(poolBalance, "gwei"),
-        mockSohm: ethers.utils.formatUnits(mockSohmBalance, "gwei"), // delete before mainnet
```

Comment:
Not sure if we should be removing this right now?

Reply:
probably can. the new network handling branch will overwrite it anyway